src/main/webapp/app/shared/util/print.service.ts

```diff
@@ -4,14 +4,11 @@ import { BehaviorSubject } from "rxjs";
 @Injectable({ providedIn: 'root' })
 export class PrintService {
 
-    public isPrintLocked$:  BehaviorSubject<boolean> = new BehaviorSubject<boolean>(false);
-
-    constructor(
-    ) {
-        this.isPrintLocked$.subscribe();
-    }
+    isPrintLocked$:  BehaviorSubject<boolean> = new BehaviorSubject<boolean>(false);
 
     setPrintLockTo(setTo: boolean) {
-        this.isPrintLocked$.next(setTo)
+        setTimeout(() => {
```

Comment:
this timeout should not be necessary, is it?

Reply:
You are right, it is not.

thought I checked that thoroughly, my bad.